SpinCore_pp/ppg/echo.py

```diff
@@ -112,10 +112,10 @@ def run_spin_echo(
             [
                 ("phase_reset", 1),
                 ("delay_TTL", deblank_us),
-                ("pulse_TTL", prog_p90_us, "ph1", ph1_cyc),
+                ("pulse_TTL", prog_p90_us.item(), "ph1", ph1_cyc),
                 ("delay", tau_us),
                 ("delay_TTL", deblank_us),
-                ("pulse_TTL", prog_p180_us, "ph2", ph2_cyc),
+                ("pulse_TTL", prog_p180_us.item(), "ph2", ph2_cyc),
```

Comment:
Don't do this here. Fix in the function that determines the programmed pulse length. If the length of the returned value is one, then call item. 

This has to do with some recent updates that I made to extend the range of applicability of that function